core/src/main/scala/kafka/admin/AdminUtils.scala

```diff
@@ -669,6 +671,30 @@ object AdminUtils extends Logging with AdminUtilities {
     props
   }
 
+  /**
+   * Read the entity (topic, broker, client, user or <user, client>) config (if any) from zk
+   * sanitizedEntityName is <topic>, <broker>, <client-id>, <user> or <user>/clients/<client-id>.
+   */
+  def fetchEntityConfigZkClient(zkClient: KafkaZkClient, rootEntityType: String, sanitizedEntityName: String): Properties = {
```

Comment:
This already exists in AdminZkClient.fetchEntityConfig().

Reply:
That method doesn't take KafkaZkClient

Reply:
AdminZkClient.fetchEntityConfig() take zkUtils.
I want to see if KafkaZkClient can be used.

Trying to figure out how to make ReplicaManagerTest pass.

Reply:
`AdminZkClient` takes `KafkaZkClient` in the constructor.

Reply:
Pardon.
AdminZkClient is not referenced by Partition or ReplicaManagerTest.
Do you mind explaining a bit more ?

Reply:
As Jun said, there's no need to add this method. `AdminZkClient.fetchEntityConfig` should be used instead. You said that the method did not take a `KafkaZkClient`, so I explained that the constructor of the class did. Can you clarify what is unclear?